clang/test/dpct/driver-stream-and-event.cu

```diff
@@ -123,3 +123,9 @@ void test_stream() {
   // CHECK: dpct::get_current_device().destroy_queue(hStream);
   cuStreamDestroy(hStream);
 }
+
+void test_cuEventRecord_crash(uint64_t hEvent,uint64_t hStream )
+{
+  // CHECK: int result = DPCT_CHECK_ERROR(*(CUevent)hEvent = (CUstream)hStream->ext_oneapi_submit_barrier());
```

Comment:
The type of the C-style cast also needs to be migrated.

Reply:
Done